util/util.go

```diff
@@ -9,3 +19,84 @@ func CloneRows(rows []parquet.Row) []parquet.Row {
 	}
 	return rr
 }
+
```

Comment:
```suggestion
// OpenParquetFiles opens the labels and chunks files, reading their magic bytes 
// and footers to validate that the files are properly formatted. 
// It is the caller's responsibility to close the returned file descriptors.
```

Reply:
I actually wanna set the "read magic flags" flag to false... :P 

This cause 2 or 3 extra calls to the object store to open the file. 

WDYT?

Reply:
Let's set it to false !

Reply:
``
It is the caller's responsibility to close file descriptors when using the files.
``

I dont think this applies here? as we do not receive a file descriptor but a `objstore.Bucket`, right?

Reply:
You are right, I thought  originally we would have to do lf.Close() but after looking deeper i realized not 👍